packages/react/src/lib/accessibility/Behaviors/Checkbox/checkboxBehavior.ts

```diff
@@ -0,0 +1,32 @@
+import * as keyboardKey from 'keyboard-key'
+import { Accessibility } from '../../types'
+
+// TODO: use after https://github.com/stardust-ui/react/pull/1421
+// type CheckboxBehaviorProps = {
+//   checked: boolean
+// }
+
+/**
+ * @specification
+ * Adds role='checkbox'. This allows screen readers to handle the component as a checkbox button.
+ * Adds attribute 'aria-checked=true' based on the property 'checked'.
+ * Adds attribute 'tabIndex=0' to 'root' component's part.
+ */
+const checkboxBehavior: Accessibility = props => ({
+  attributes: {
+    root: {
+      'aria-checked': props.checked,
+      role: 'checkbox',
+      tabIndex: 0,
```

Comment:
Just to be sure, we should not add any attribute if the checkbox is disabled?

Reply:
https://www.w3.org/TR/wai-aria-practices/examples/checkbox/checkbox-1/checkbox-1.html

I don't see this in the spec, so decided to skip for the initial implementation